bin/vaultclient

```diff
@@ -98,28 +100,13 @@ program
     .command('create-account')
     .option('--name <NAME>')
     .option('--email <EMAIL>')
-    .option('--password <PASSWORD>')
     .action(action.bind(null, 'create-account', (client, args) => {
         client.createAccount(
-            args.name, { email: args.email, password: args.password },
+            args.name, { email: args.email },
             handleVaultResponse);
     }));
 
 program
-    .command('list-account-users')
```

Comment:
There are three different listing commands in vaultclient:
-list-accounts: not present in AWS API, neither in aws cli
-list-account-users
-list-access-keys

From those three we should keep list-accounts as it's requested by CSEs and not available in the official tools.